CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Decidim::Organization.find_each { |organization| Decidim::System::CreateDefaultP
 
 **Added**:
 
+**decidim-core**: Add readonly attribute to date_fields so that the user is forced to use the datepicker. [#3705](https://github.com/CodiTramuntana/decidim/pull/3705)
```

Comment:
the url is correct? should not be ["https://github.com/decidim/decidim/pull/3705"](https://github.com/decidim/decidim/pull/3705) ?